specs/marketplace.md

```diff
@@ -109,29 +103,24 @@ Time ran out │ │                              └─────────
 
 ![image](./images/storeRequest.png)
 
-## Client role
+## Client Role
 
-The Client role represents nodes that introduce data to be persisted inside the Codex network. 
+A node implementing the Client role mediates the persistence of data within the Codex network.
 
-There are two parts of a client role:
+A Client has two primary responsibilities:
 
- - Requesting storage from the network by creating a storage request.
- - Withdrawing funds from storage requests.
+- Requesting storage from the network by sending a Storage Request to the Smart Contract.
+- Withdrawing funds from the Storage Requests previously created by the Client.
 
-### Creating storage requests
+### Creating Storage Requests
 
-When the user prompts the client node to create a storage request, 
-it SHOULD receive the input parameters for the storage request from the user. 
+When a user prompts the Client node to create a Storage Request, the Client node SHOULD receive the input parameters for the Storage Request from the user.
 
-To create a request to persist a dataset on the Codex network,
-client nodes MUST split the dataset into data chunks, $(c_1, c_2, c_3, \ldots, c_{n})$.
-Using an erasure coding technique and input parameters, the data chunks are encoded and placed into separate slots.
-The erasure coding technique MUST be the [Reed-Soloman algorithm](https://hackmd.io/FB58eZQoTNm-dnhu0Y1XnA). 
-The final slot's roots and other metadata MUST be placed into a Manifest (**TODO: Manifest RFC**). The manifest's CID MUST then 
-be used as the `cid` of the stored dataset.
+To create a request to persist a dataset on the Codex network, Client nodes MUST split the dataset into data chunks, $(c_1, c_2, c_3, \ldots, c_{n})$. Using the Erasure Coding method and the provided input parameters, the data chunks are encoded and distributed over a number of slots. The applied Erasure Coding method MUST use the [Reed-Soloman algorithm](https://hackmd.io/FB58eZQoTNm-dnhu0Y1XnA). The final slot roots and other metadata MUST be placed into a Manifest (TODO: Manifest RFC). The CID for the Manifest MUST then be used as the `cid` for the stored dataset.
```

Comment:
```suggestion
To create a request to persist a dataset on the Codex network, client nodes MUST split the dataset into data chunks, $(c_1, c_2, c_3, \ldots, c_{n})$. Using the erasure coding method and the provided input parameters, the data chunks are encoded and distributed over a number of slots. The applied erasure coding method MUST use the [Reed-Soloman algorithm](https://hackmd.io/FB58eZQoTNm-dnhu0Y1XnA). The final slot roots and other metadata MUST be placed into a `Manifest` (TODO: Manifest RFC). The CID for the `Manifest` MUST then be used as the `cid` for the stored dataset.
```

Here, again, "manifest" is only a proper noun if it describes a very specific, unique object, like an object structure defined in this document or in another document. I made the assumption that this is going to describe such an object _in the future_.

Reply:
I was initially thinking we should keep Erasure Coding capitalised, but now I think you are right - it is standard generic think. If we speak about proper Erasure Coding method that has its own name, than we would introduce it as capitalised. Somehow I though Erasure Coding also deserves that... I will correct it.